src/actions/arcActions.ts

```diff
@@ -711,9 +712,9 @@ export function stakeProposal(daoAvatarAddress: string, proposalId: string, pred
       const stakingToken = await StandardToken.at(await votingMachineInstance.contract.stakingToken());
       const balance = await stakingToken.balanceOf(currentAccountAddress);
 
-      const amount = web3.toWei(stake, "ether");
-      if (amount < minimumStakingFee) { throw new Error(`Staked less than the minimum: ${minimumStakingFee}!`); }
-      if (amount > balance) { throw new Error(`Staked more than than the balance: ${balance}!`); }
+      const amount: BigNumber = new BigNumber(web3.toWei(stake, 'ether'));
+      if (amount.lt(minimumStakingFee)) { throw new Error(`Staked less than the minimum: ${Number(web3.fromWei(minimumStakingFee, 'ether'))}!`); }
```

Comment:
I created a function Util.fromWei that casts to a number and doesnt requite passing in 'ether'